### ISSUE TITLE
demo link is dead

### ISSUE THREAD

**Issue by Archer-Thane** [icebob/kantab#6]

**Comment by icebob** [comment]
Yeah, I know just Vercel dropped docker support and only support serverless projects, and I don't find any free alternative yet to deploy a demo.

**Comment by Archer-Thane** [comment]
Hey, I have a dedicated server which can share with you for the demo. if you interested email me: 
bahrami.ehssan@gmail.com
